platforms/web/lib/useListeners/utils.ts

```diff
@@ -14,14 +14,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { FORMATTING_ACTIONS } from '../constants';
-import { FormattingState, FormattingActions } from '../types';
+import { ACTION_TYPES } from '../constants';
+import { ActionTypes, ActionState, AllActionStates } from '../types';
 
-export function getDefaultFormattingStates() {
-    return FORMATTING_ACTIONS.reduce<
-        Record<FormattingActions, FormattingState>
-    >((acc, action) => {
+export function defaultActionStates(): AllActionStates {
```

Comment:
I prefer when the functions have a verb in their name to be more explicit

Reply:
Yes but syntactically it's a function and you can't use it as `const myState = defaultActionStates`

Reply:
Cool, renamed to createDefaultActionStates